apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -69,53 +75,32 @@ public void Dispose()
             await FillSampleData();
         }
 
-        private async Task<Database> CreateDatabaseAsync()
+        private async Task CreateDatabaseAsync()
         {
-            // Create client
-            _databaseAdminClient = await DatabaseAdminClient.CreateAsync();
-            // Initialize request argument(s)
-            var parent = new InstanceName(TestProjectName, TestInstanceName);
-
-            string createStatement = "CREATE DATABASE " + DatabaseName;
-
-            // Make the request
-            var response = await _databaseAdminClient.CreateDatabaseAsync(parent, createStatement);
-
-            // Poll until the returned long-running operation is complete
-            var completedResponse = await response.PollUntilCompletedAsync();
-
-            // Retrieve the operation result
-            return completedResponse.Result;
+            using (var connection = new SpannerConnection(NoDbConnectionString))
+            {
+                var createCmd = connection.CreateDdlCommand("CREATE DATABASE " + DatabaseName);
+                await createCmd.ExecuteNonQueryAsync(CancellationToken.None);
+            }
         }
 
         private async Task CreateTxTableAsync()
         {
-            // Create client
-            var databaseAdminClient = await DatabaseAdminClient.CreateAsync();
-
             var typeTable = @"CREATE TABLE TX (
                               K                   STRING(MAX) NOT NULL,
                               Int64Value          INT64,
                               StringValue         STRING(MAX),
                             ) PRIMARY KEY (K)";
-            // Make the request
-            var response =
-                await databaseAdminClient.UpdateDatabaseDdlAsync(
-                    new DatabaseName(
-                        TestProjectName,
-                        TestInstanceName,
-                        DatabaseName),
-                    new[] {typeTable});
-
-            // Poll until the returned long-running operation is complete
-            await response.PollUntilCompletedAsync();
+
+            using (var connection = new SpannerConnection(ConnectionString))
```

Comment:
Given the repetition of this block, maybe extract to a method of `ExecuteDdlAsync(string ddl)`?